packages/core/src/browser/shell/tab-bar-toolbar/tab-bar-toolbar.tsx

```diff
@@ -139,10 +164,10 @@ export class TabBarToolbar extends ReactWidget {
         </div>;
     }
 
-    protected getToolbarItemClassNames(commandId: string | undefined): string[] {
+    protected getToolbarItemClassNames(commandId: string | undefined, whenClauseResult: boolean): string[] {
```

Comment:
We should probably pass `item: AnyToolBarItem` instead of `commandId` and `whenClauseResult`.

Reply:
but if we pass in just the item wouldn't we have to do the command extraction at line 144 again?

Reply:
> but if we pass in just the item wouldn't we have to do the command extraction at line 144 again?

No, we would just use `item.command` and `item.when`?

Reply:
oh ok. Im sorry i didn't realize that `item.command` and `command.id` would be the same value. I'll do that. thanks for clarifiying